ocs_ci/deployment/cnv.py

```diff
@@ -0,0 +1,511 @@
+        csv_obj.wait_for_phase(phase="Succeeded", timeout=720)
+
+    def wait_for_the_resource_to_discover(self, kind, namespace, resource_name):
+        """
+        Waits for the specified resource to be discovered.
+
+        Args:
+            kind (str): The type of the resource to wait for.
+            namespace (str): The namespace in which to wait for the resource.
+            resource_name (str): The name of the resource to wait for.
+
+        """
+        logger.info(f"Waiting for resource {kind} to be discovered")
+        for sample in TimeoutSampler(300, 10, ocp.OCP, kind=kind, namespace=namespace):
+            resources = sample.get().get("items", [])
+            for resource in resources:
+                found_resource_name = resource.get("metadata", {}).get("name", "")
+                if resource_name in found_resource_name:
+                    logger.info(f"{kind} found: {found_resource_name}")
+                    return
+                logger.debug(f"Still waiting for the {kind}: {resource_name}")
+
+    def deploy_hyper_converged(self):
+        """
+        Deploys the HyperConverged CR.
+
+        Raises:
+            TimeoutExpiredError: If the HyperConverged resource does not become available within the specified time.
+
+        """
+        logger.info("Deploying the HyperConverged CR")
+        hyperconverged_yaml_file = templating.load_yaml(CNV_HYPERCONVERGED_YAML)
+        hyperconverged_yaml = OCS(**hyperconverged_yaml_file)
+        hyperconverged_yaml.create()
+
+        # Verify the installation was completed successfully by checking the HyperConverged CR
+        ocp = OCP(kind=constants.HYPERCONVERGED, namespace=self.namespace)
+        # Wait for the HyperConverged resource named 'kubevirt-hyperconverged'
+        # to be in the 'Available' condition within a timeout of 600 seconds.
+        result = ocp.wait(
+            resource_name=constants.KUBEVIRT_HYPERCONVERGED,
+            condition="Available",
+            timeout=600,
+        )
+        if not result:
+            err_str = (
+                "Timeout occurred, or the HyperConverged resource did not become available "
+                "within the specified time."
+            )
+            raise exceptions.TimeoutExpiredError(err_str)
+        logger.info(
+            f"{constants.HYPERCONVERGED} {constants.KUBEVIRT_HYPERCONVERGED} met condition: Available"
+        )
+
+    def enable_software_emulation(self):
+        """
+        Enable software emulation. This is needed on a cluster where the nodes do not support hardware emulation.
+
+        Note that software emulation, when enabled, is only used as a fallback when hardware emulation
+        is not available. Hardware emulation is always attempted first, regardless of the value of the
+        useEmulation.
+
+        """
+        if (
+            config.ENV_DATA["platform"].lower() == "baremetal"
+            and config.DEPLOYMENT.get("local_storage") is True
```

Comment:
@PrasadDesala , I guess 
```suggestion
            and config.DEPLOYMENT.get("local_storage")
``` is enough to check the truth value